recipes/coost/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
It can be removed. Using new CMake tools, the cmake wrapper is no longer used.

Reply:
Remove the whole CMakeLists.txt?

Reply:
Yes. It's no longer used. Conan will generate a cmake toolchain file, which will be used to inject settings, options and recipe variables. It replaces that CMakelists.txt wrapper.